Kernel/Bus/USB/UHCI/UHCIController.cpp

```diff
@@ -309,20 +308,12 @@ KResult UHCIController::reset()
     return KSuccess;
 }
 
-UNMAP_AFTER_INIT void UHCIController::create_structures()
+UNMAP_AFTER_INIT bool UHCIController::create_structures()
```

Comment:
Since the caller returns `KResult`, should we just return `KResult` here as well? That would theoretically enable us to return "richer" error codes if the case ever arose...